weblate/checks/chars.py

```diff
@@ -39,6 +39,23 @@
 FRENCH_PUNCTUATION = {";", ":", "?", "!"}
 
 
+class AcceleratorKeyCheck(TargetCheck):
+    """Check for inconsistent accelerator keys."""
+
+    check_id = "accelerators"
+    name = _("Accelerator key")
+    description = _("Source and translation do not both contain an accelerator key")
+    default_disabled = True
+
+    def _check_accelerator(self, source, target, key):
+        return (target.count(key) > 1) or (source.count(key) != target.count(key))
+
+    def check_single(self, source, target, unit):
+        return self._check_accelerator(source, target, "&") or self._check_accelerator(
+            source, target, "_"
+        )
```

Comment:
Gettext checks for `&` only (unless something else is specified on the command line).

Reply:
I was checking the interwebs for different accelerators and came across this: http://docs.translatehouse.org/projects/localization-guide/en/latest/guide/translation/accelerators.html#accelerators. I knew only `&` and `_` until now, where `&`is the most common. We could add `~` too and always check for all three or we could add an `accelerator_keys` setting, make `&` the default and let the user override it. But would that setting be global, per project or even per component?
How big would the risk for false positives be, if we always check for all three?

Reply:
The flag can take argument as well. So `accelerator` would check for `&` and `accelerator:_` for `_`.

Reply:
is there an example somewhere how flags with arguments work?

Reply:
For example the PlaceholderCheck:

- you subclass from TargetCheckParametrized
- param_type property defines parsing (as this needs a string, `return single_value_flag(lambda x: x)` should do it)
- you define check_target_params instead of check_single, it get's parameter value as last arg
- I'm not sure if this can gracefully handle defaults when no param is specified though...

Reply:
Thanks, I'll take a look